mountd/rpc.c

```diff
@@ -1002,9 +1116,17 @@ static int dcerpc_bind_ack_return(struct ksmbd_rpc_pipe *pipe)
 	if (dce->bi_req.assoc_group_id == 0)
 		dce->bi_req.assoc_group_id = 0x53f0;
 
-	ndr_write_int16(dce, dce->bi_req.max_xmit_frag_sz);
-	ndr_write_int16(dce, dce->bi_req.max_recv_frag_sz);
-	ndr_write_int32(dce, dce->bi_req.assoc_group_id);
+	ret = ndr_write_int16(dce, dce->bi_req.max_xmit_frag_sz);
```

Comment:
We need ret variable ? how is this ?
```
if (ndr_write_int16(dce, dce->bi_req.max_xmit_frag_sz))
     return KSMBD_RPC_EBAD_DATA;
```

Reply:
I started with having ret variable and then copied the same pattern pretty much everywhere. Do you want me to change the occurrences to what you suggest instead ?